lib/jsonLayout.js

```diff
@@ -79,6 +79,13 @@ function jsonLayout(config) {
 
     _.assign(output, overlays);
 
+    // Add appName if specified
+    if (_.has(config, 'appName')) {
```

Comment:
do you mind moving this to line 64, just after the `config.source` handling? it will be clearer what gets added.
thanks